delocate/tools.py

```diff
@@ -519,6 +521,25 @@ def _get_install_ids(filename: str | PathLike[str]) -> dict[str, str]:
     return out
 
 
+@ensure_writable
+def _set_install_names(
+    path: str | PathLike[str], changes: Iterable[tuple[str, str]]
+) -> None:
+    """Set install names in binary `path`.
+
+    Parameters
+    ----------
+    path
+        Path to binary file.
+    changes
+        (old, new) pairs
+    """
+    options: list[str] = []
+    for oldname, newname in changes:
+        options.extend(("-change", oldname, newname))
+    _run(["install_name_tool", *options, str(Path(path))], check=True)
```

Comment:
Path can used directly as subprocess arguments.
```suggestion
    _run(["install_name_tool", *options, Path(path)], check=True)
```